### DISCUSSION TITLE
Add events for pull request review comment resolved/unresolved

### DISCUSSION THREAD

**Discussion by jackbentley** [community/community#12811] [Actions]
I'm trying to integrate into another system via GitHub actions/workflows. I want a specific workflow to execute when there are a certain number of approved reviews and only when all comments have been resolved.

The former seems to work fine as I'm able to use the `pull_request_review ` event and `submitted` action type. However, no such event or action exists for when a comment is marked as resolved/unresolved. I would have expected there to have been another action type for the `pull_request_review_comment` event.

**Comment by ethomson** [comment]
Hi @jackbentley - we appreciate the feedback. I'll flag this to the team that works on triggers to see if this is something that they can put on their backlog.

Thanks again!

**Reply by jackbentley** [reply]
Looks like this is already implemented in webhooks with `pull_request_review_thread` but has not been ported over to workflows?

https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request_review_thread

**Reply by carlocab** [reply]
Gentle nudge -- would be great if this were implemented!

**Comment by RikSchefferAmsterdam** [comment]
I would love to have this

**Comment by alexipatova** [comment]
Would love to have this as well! ✨

**Comment by benotoho** [comment]
I've been waiting for this as well. 🙂

**Comment by zaniastone** [comment]
Ah, I was so hopeful when I found this issue from 4 years ago. :(

**Comment by AStarchikov** [comment]
Would be great to have such

**Comment by jvonoetinger-productdna** [comment]
+1
